### ISSUE TITLE
microcopy for top links on topic page

### ISSUE THREAD

**Issue by briaguya** [cityofaustin/techstack#3265]
**Acceptance criteria**
Microcopy for picker for top links on a topic page read:
- [x] title: `Choose content to feature on this topic`
- [x] helper text: `Only the following published page types within this topic may be chosen for this field: Information page, Service page, Guide page, Official document page. Search results will exclude pages of other types.`

--------

@easherma re: @desigonz's microcopy changes:

looks like the logic for that line is happening in https://github.com/cityofaustin/joplin/blob/cf0c03cb25ba7f26cfff5ca038a4498f466e7329/joplin/templates/wagtailadmin/chooser/browse.html#L11

I'd consider this a bug in how we're determining the choose_str logic there, as there will be non department situations where we want to choose all 4 non IA page types.

_Originally posted by @briaguya in https://github.com/cityofaustin/techstack/issues/3188#issuecomment-546056760_

> ### Choosing a top links on a topic
> * [ ]  Need to update the microcopy:
>   
>   * [ ]  `Choose content to feature on this topic`
>   * [ ]  `Only the following published page types within this topic may be chosen for this field: Information page, Service page, Guide page, Official document page. Search results will exclude pages of other types.`
> 
> ![image](https://user-images.githubusercontent.com/5839508/67514094-bde9fd80-f661-11e9-8422-9aaafaa3107d.png)

**Comment by easherma** [comment]
This could be the straw that breaks the camel's back in terms of us finally just writing our own choosers using wagtail-autocomplete or wagtail-generic-chooser

**Comment by easherma** [comment]
I haven't found any way as-is to tell these choosers apart with the context provided. 

Potential options: 
* Might be able to make a context processor that passes the model of the current page to the chooser
* Might be able to use a template tag that something similar

I tried both options briefly, but didn't get anywhere, and to be honest I'm not positive they can even be used in such a way.

**Comment by easherma** [comment]
None of the above options work. 

Neither generic-chooser nor wagtail-autocomplete take a list of models; they only expect one. 

I've been playing around with deeply extending some wagtail core stuff. I've learned alot, but I haven't come up with anything super useful yet. 

Starting to question if this is a good use of time to try and implement vs just using more generic copy when you're choosing just about any page to link to (currently relating pages to departments and top links to topics). 

The need to even link the above two use cases as is is a limitation of Joplin Alpha. There could be ways in Joplin 3.0 to make this _way_ easier on authors/admins (e.g following a traditional page hierarchy with the option for linking the page from multiple other places)

**Comment by easherma** [comment]
https://austininnovation.slack.com/archives/CPZ16AY07/p1573509702004700

**Comment by desigonz** [comment]
Pasting in @easherma's linked comments for posterity/ease of access:

Awhile back we added custom copy for page choosers that involved some tech debt.
Now we have what we expected would happen; two page choosers that have the same range of page types, so they have the same copy.
In this case it is:
• top services on a department page
• top links on a topic page
The only difference between the two is:
Choose content to feature on the department page vs
Choose content to feature on this topic
The list of acceptable page types is the same, for now.
Would either of the following be an acceptable solution for now? (pre Joplin 3.0):
1) Change the copy for both cases to read: Chose content to feature on this page
2) Make the list of acceptable content types slightly different so they can be told apart, such as: (edited) 
Choose content to feature on the department page

Only the following published page types may be chosen for this field: Information page, Service page, Guide page, Official document page. Search results will exclude pages of other types.
Choose content to feature on this topic
Only the following published page types may be chosen for this field: Service page, Information page, Guide page, Official document page. Search results will exclude pages of other types.
( I just swapped the order of Service page and Info page on the list )
Option 3) Keep poking with it, try to get a long term solution. This frankly is going to involve some deep hacks to wagtail or extending a plugin that isn't well supported, may many more days to implement, and there is a good chance we might change this interface of selecting related pages anyways since we don't really like it overall. I'm not unwilling to do it, but there may be other stuff more valuable for us to work on (since all we are currently getting here is a title change inside a modal).
